tests/run.go

```diff
@@ -205,11 +205,16 @@ func Run(o *option.Option) {
 		})
 
 		ginkgo.It("should share PID namespace with host with --pid=host", func() {
```

Comment:
I think we need to verify the container has picked up the PID in the host operating system. I haven't came up with a good solution for it, if it's blocking PR maybe we can remove it for now and add a TODO? WDYT @AnqiPang @davidhsingyuchen ?

Reply:
Maybe that's (partly) addressed by https://github.com/runfinch/common-tests/pull/8#discussion_r1034211237?